cmd/av/pr_queue.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine 	bool
+	Targets		string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pr for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		variables, err := getMutationVariables()
+		if err != nil {
+			return err
+		}
+
+
+		// I have a feeling this would be better written inside of av/internals
+		client := avgql.NewClient()
+
+		var mutation struct {
+			QueuePullRequest struct {
+				UpdateMergeQueueStatusPayload struct {
+					PullRequest	graphql.String
+				}
+					
+			} `graphql:"githubRepository(input: {owner: $repoOwner, name:$repoName})"`
+		}
+		err = client.Mutate(context.Background(), &mutation, variables)
+		if err != nil {
+			return err
+		}
+		// We print to stderr everywhere, why?
```

Comment:
We print to stderr because we don't want users to rely on the output of `stdout`. imo all logging should go to stderr and stdout should only be used for programatic output.

Reply:
Done